CODEOWNERS

```diff
@@ -13,4 +13,6 @@
 gradle* @planetf1
 gradle/ @planetf1
 SECURITY.md @planetf1
-Dockerfile @planetf1
+src/ @juergenhemelt
```

Comment:
github is showing this file as containing errors. I think it's complaining about the github id. Probably as you are not a maintainer. will come back to this

Reply:
Checked - I had not added you as maintainer for this repo. will do so and check the review again.

Reply:
Now showing as good so closing this comment